app/models/item.rb

```diff
@@ -135,19 +135,23 @@ def has_inventory?(inventory = nil)
     end
   end
 
-  def is_in_kit?
-    organization.kits
-      .active
```

Comment:
The controller definition of `@kits` doesn't narrow it down to `.active` the way this does (did)